doc-content/enterprise-only/bpmn/bpmn-subprocesses-ref.adoc

```diff
@@ -25,14 +25,26 @@ An embedded subprocess encapsulates a part of the process. It must contain a sta
 
 An ad hoc subprocess or process contains a number of embedded inner activities and is intended to be executed with a more flexible ordering compared to the typical routing of processes. Unlike regular processes, an ad hoc subprocess does not contain a complete, structured BPMN2 diagram description, for example, from start event to end event. Instead, the ad hoc subprocess contains only activities, sequence flows, gateways, and intermediate events. An ad hoc subprocess can also contain data objects and data associations. The activities within the ad hoc subprocesses are not required to have incoming and outgoing sequence flows. However, you can specify sequence flows between some of the contained activities. When used, sequence flows provide the same ordering constraints as in a regular process. To have any meaning, intermediate events must have outgoing sequence flows and they can be triggered multiple times while the ad hoc subprocess is active.
 
-
 .Reusable subprocess
-Reusable subprocesses appear collapsed within the containing process.
+Reusable subprocesses appear collapsed within the parent process. Configure the reusable subprocess using the following  properties by clicking image:getting-started/diagram_properties.png[] to open the *Properties* panel and expand *Implementation/Execution*.
+
+* *Called Element*: The ID of the subprocess that is called and instantiated by the activity.
+* *Independent*: If selected, the subprocess is started as an independent process, which means that the subprocess is not terminated if the parent process is completed or if the subprocess is canceled. If not selected, the active subprocess is canceled when the parent process is terminated or when the subprocess is canceled. This property is only visible when the *Wait for completion* property is selected.
+* *Abort Parent*: If selected, non-independent, reusable subprocesses can abort the parent process when there's an error during the execution of the called process instance. For example, when there's an error when trying to invoke the subprocess or when the subprocess instance is aborted. This property is only visible when the *Independent* property is not selected. The following rules apply:
+** If the reusable subprocess is independent, *Abort parent* is not available.
```

Comment:
This statement and the one after that duplicate the statement in the previous paragraph:  This property is only visible when the *Independent* property is not selected.

I don't mind idiot-proofing the doc but I suspect this passage looks a bit too obvious as idiot-proofing. Your call at the end of the day.

Reply:
I agree. Spent more time on getting the info corrected with dev and missed the redundancy. Updated. Thank you.